neuraxle/steps/misc.py

```diff
@@ -378,10 +389,16 @@ def _fit_transform_data_container(self, data_container: DataContainer, context:
         self.handle_fit_transform_callback((data_container, context))
         return self, data_container
 
+    def fit(self, data_inputs, expected_outputs) -> '_FittableStep':
+        return self
+
+    def inverse_transform(self, processed_outputs):
```

Comment:
seems like those two methods could be provided by default by something to inherit from.